### PR TITLE
Adding Hover effect

### PR DESCRIPTION
**Pull request by shwetazade27**
## Related Issue
#120 

## Description of Changes

Added hovering effect on navbar tabs 

## Screenshots
original:
old_one
updated:
new_one

### CODE REVIEW

**Comment by anmode** [comment]
please update your pR. fetch the latest changes.

**Comment by shwetazade27** [comment]
@anmode sir please help me with this one how to update pr

**Comment by anmode** [comment]
follow the steps:
1. Checkout the develop branch ( `git checkout develop`)
2. Pull the latest changes ( `git pull` )
3. try one more time `git fetch`
make sure your remote (your github repo) is upto date with usptream(my github repo)
4. checkout to your feature branch (`git checkout your_branch`)
5. `git merge develop`

**Comment by shwetazade27** [comment]
Done sir... @anmode 
updated

**Comment by anmode** [comment]
great!

File: styles/LoginDropdown.module.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.navLink:hover {
+  color:rgb(234, 185, 239) !important;
+  transform: scale(1.1); 
+}
```
Comment (anmode) [inline styles/LoginDropdown.module.css:4]: Tbh I do not like the hover...please check out https://www.amupedia.com/. see hover effect

**Review by anmode — CHANGES_REQUESTED** [review]
One more things please see and optimize for mobile too!

**Comment by anmode** [comment]
I did not like the color...btw can you also change the footer hyperlink...UI. As of now line is coming we can have any effect

**Comment by shwetazade27** [comment]
@anmode sir I have made bit changes in color and in  footer links please let me know if further changes or improvement is needed

**Comment by anmode** [comment]
> @anmode sir I have made bit changes in color and in footer links please let me know if further changes or improvement is needed

Shweta...One thing you can do.. Just made it like...when we hover it glows. is that ok? after this change I'll merge thanks

**Comment by shwetazade27** [comment]
@anmode done.. please let me know if changes or improvement is needed.

**Comment by anmode** [comment]
It should not glow initially! Make it like login button but afterward on hover it should glow

**Comment by anmode** [comment]
See in production as of now www.grabtern.com now by hovering it should glow that it is!!

**Comment by shwetazade27** [comment]
@anmode done the changes, please let me know if further change is required. Thanks!

**Review by anmode — APPROVED** [review]
great! just run now `npx prettier --write .`

**Comment by shwetazade27** [comment]
@anmode done

**Comment by anmode** [comment]
congrats @shwetazade27

**Pull request merged**